Packs/PaloAltoNetworks_IoT3rdParty/Playbooks/Incremental_Export_Devices_to_ServiceNow_-_PANW_IoT_3rd_Party_Integration_README.md

```diff
@@ -0,0 +1,49 @@
+This playbook should be used in a recurring Job to get incremental updates for devices to send to ServiceNow server.
```

Comment:
Gets updated devices and sends the updates to ServiceNow. You should run this playbook as a scheduled, recurring job. 

{I'd give a real-world example of when they would schedule the job for.)